### ISSUE TITLE
docker for Windows 10 login failure/timeout

### ISSUE THREAD

**Issue by benjaminellison** [moby/moby#31097]
When issuing a login command (via Powershell ISE run as Administrator) with explicitly defined parameters:
`docker login -u myusername -p mysecretword https://hub.docker.com`
I receive the following response:
```
docker : Error response from daemon: Get https://hub.docker.com/v1/users/: dial tcp: lookup hub.docker.com on 192.168.65.1:53: read udp 
192.168.65.2:36833->192.168.65.1:53: i/o timeout
At line:1 char:1
+ docker login -u myusername -p mysecretword https://hub.docker.co ...
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : NotSpecified: (Error response ...53: i/o timeout:String) [], RemoteException
    + FullyQualifiedErrorId : NativeCommandError
```

I'm running Windows 10 Enterprise, Hyper-V, and docker version output is:
```
PS C:\Windows\system32> docker version
Client:
 Version:      1.13.1
 API version:  1.26
 Go version:   go1.7.5
 Git commit:   092cba3
 Built:        Wed Feb  8 08:47:51 2017
 OS/Arch:      windows/amd64

Server:
 Version:      1.13.1
 API version:  1.26 (minimum version 1.12)
 Go version:   go1.7.5
 Git commit:   092cba3
 Built:        Wed Feb  8 08:47:51 2017
 OS/Arch:      linux/amd64
 Experimental: true
```

I've tried changing the DNS in the Docker->Network settings to from Automatic to Fixed (8.8.8.8). No luck.  
I tried setting the DockerNAT in the Hyper-V Virtual Switch Manager to "External" via my NIC, with no luck.
I tried adding hub.docker.com:5000 to the insecure registry in Docker->Daemon settings, with no luck.

If I don't specify all three items (username, password, and a location), it just hangs (appears to be the same as issue #15941).  If I do specify them, I get the timeout error.  BTW, the 192.168.x.x isn't a space my machine is on, nor can I see that anywhere else (the Docker->Network settings say it is at 10.0.75.X).  

I've done quite a bit of searching, and it appears there was an earlier issue related to boot2docker (which I'm not using) that was closed, but nothing seems to be currently tracking this with a viable solution.

**Comment by friism** [comment]
@benjaminellison do you also get this error if you use a non-ISE powershell or cmd.exe?

**Comment by benjaminellison** [comment]
I just checked and I do indeed still receive that error in those cases. I've tried all four variations of PS and PS ISE both with and without Administrator. The error message returned only varied in verbosity (a one line timeout message versus the extended one I indicate above).
For giggles and grins, anticipating it wouldn't make any difference, I tried it with a made-up username/pw combo - and was correct. The error above still happens.
I also tried it with a made-up repository (https://crazymadeup.repo.com instead of https://hub.docker.com) -- same exact timeout error.
